src/libos/crates/async-rt/src/scheduler/local_scheduler/mod.rs

```diff
@@ -241,36 +248,37 @@ impl<E: SchedEntity> LocalScheduler<E> {
         num_drained
     }
 
-    /// The running vcpu wait until being enqueued tasks
-    pub fn wait_enqueue(&self) {
+    /// The running vcpu keep idle state until being enqueued tasks.
+    /// Return true if tasks has been enqueued in idle time.
+    /// Return false if idle time has been depleted.
+    pub fn wait_idle(&self) -> bool {
```

Comment:
I still think it is better to merge `wait_idle` and `wait_sleep` into wait_equeue.

Reply:
Done